components/wizard/structure/content.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Content
+page_title: Wizard Content
+description: Content of the Wizard for Blazor.
+slug: wizard-structure-content
+tags: telerik,blazor,wizard,content
+published: True
+position: 3
+---
+
+# Wizard Content
+
+The `Content` of the wizard is a render fragment of the `WizardStep`. While it is a common use case to integrate a form in the Wizard Content, it can also contain any type of HTML content including various Telerik components as needed to cover your desired scenario.
```

Comment:
I do not understand the first sentence. The Content of the Wizard is a render fragment of the WizardStep. Maybe it should be **The Content is a Child content of a WizardStep**